Doc/library/asyncio-policy.rst

```diff
@@ -184,23 +186,64 @@ implementation used by the asyncio event loop:
 
       Note: loop may be ``None``.
 
+   .. method:: is_active()
+
+      Returns ``True`` is the watcher is ready to use, e.g. :py:data:`SIGCHLD`
```

Comment:
```suggestion
      Return ``True`` if the watcher is ready to use.
```

(Fix a typo; drop the "e.g." and everything after it)

Reply:
"Spawning a subprocess with *inactive* current child watcher raises :exc:`RuntimeError`." on a new line.